### DISCUSSION TITLE
Bug report: Exporting video to mp4 messes up fps

### DISCUSSION THREAD

**Discussion by AyeoJosh** [keiwando/toonsquid#370] [Q&A]
When you export to mp4 it doesn’t export to the right fps, but when you export to Mov it’s the right fps. It’s supposed to be 24.

MP4:
![IMG_0055](https://github.com/keiwando/toonsquid/assets/140559966/379ded60-0fe4-4261-85ff-e35364888886)

Mov:
![IMG_0057](https://github.com/keiwando/toonsquid/assets/140559966/9959b354-476a-4454-9f30-e6317ff82495)

**Comment by keiwando** [comment]
Hi! I am unable to reproduce this. Both the mov and mp4 export have the same frame rate for me. Does this happen in all projects for you or only in a specific one?
